src/utils/pageUtils.ts

```diff
@@ -14,16 +14,24 @@ export function getControls(): HTMLElement {
     ];
 
     for (const controlsSelector of controlsSelectors) {
-        const controls = document.querySelectorAll(controlsSelector);
+        const controls = Array.from(document.querySelectorAll(controlsSelector)).filter(el => !isInPreviewPlayer(el));
 
-        if (controls && controls.length > 0) {
+        if (controls.length > 0) {
             return <HTMLElement> controls[controls.length - 1];
         }
     }
 
     return null;
 }
 
+export function isInPreviewPlayer(element: Element): boolean {
+    while (element != null) {
+        if (element.id === "inline-preview-player") return true;
+        element = element.parentElement;
+    }
+    return false;
```

Comment:
```suggestion
    return !!element.closest("#inline-preview-player");
```

let me know if that works

Reply:
It obviously does.
i love not knowing the whole std library, then reimplementing simple things like these and finally realizing i've written redundant code